docs/orchestration/docker-compose/nginx.conf

```diff
@@ -0,0 +1,52 @@
+
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    # include /etc/nginx/conf.d/*.conf;
+
+    upstream minio {
+        server minio1:9000;
+        server minio2:9000;
+        server minio3:9000;
+        server minio4:9000;
+    }
+
+    server {
+        listen       9000;
+        listen  [::]:9000;
+        server_name  localhost;
+        location / {
+            proxy_set_header Host $host;
```

Comment:
`$host` needs to be `$http_host`

Reply:
It is more appropriate to use `$host`, because the `$http_host` in the request header may be empty. This is the official Nginx configuration of docker. @vadmeste

Reply:
> It is more appropriate to use `$host`, because the `$http_host` in the request header may be empty. This is the official Nginx configuration of docker. @vadmeste

For some reasons, $host didn't work for me, did you test your docker-compose file ?

Reply:
Yes,i use MinIO Browser to test it.Can you tell me your question? Let me try.

Reply:
> Yes,i use MinIO Browser to test it.Can you tell me your question? Let me try.

oh, you need to test with `mc` as well. 'mc' uses S3 protocol, and the latter requires the Host header to be untouched when passing through the proxy.

Reply:
Look at the right configuration here https://docs.minio.io/docs/setup-nginx-proxy-with-minio.html

Reply:
Resolved, thanks. I have used the browser and `mc` to test.